rpcs3/Emu/Cell/Modules/sceNp2.cpp

```diff
@@ -623,8 +623,17 @@ error_code sceNpMatching2SignalingGetConnectionInfo(
 			connInfo->bandwidth = 10'000'000; // 10 MBPS HACK
 			break;
 		}
-		case 5:
+	    case 4:
+	    {
+		    auto& sigh = g_fxo->get<named_thread<signaling_handler>>();
+		    const auto si = sigh.get_sig2_infos(roomId, memberId);
+		    connInfo->address.port = std::bit_cast<u16, be_t<u16>>(si.port);
+		    connInfo->address.addr.np_s_addr = si.addr;
+		    break;
+	    }
+	    case 5:
```

Comment:
this whole section seems to be using spaces instead of tabs for indentation ❗

Reply:
❗